### PR TITLE
Improve DANDI Upload to Allow for Both Staging and Main API Keys

### LINKED ISSUE #367: Upload to staging server
- [ ] Settings page should include a DANDI staging API key, which is different from production API key
- [ ] Upload page should include a checkbox for staging
image

### PR DESCRIPTION
**Pull request by garrettmflynn**
fix #367

### CODE REVIEW

File: schemas/json/dandi/upload.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
     "properties": {
         "dandiset_id": {
-            "type": "string"
+            "type": "string",
+            "description": "The unique identifier for your dandiset. Will automatically determine whether to upload to the main or staging DANDI server."
```
Comment (CodyCBakerPhD) [inline schemas/json/dandi/upload.json:5]:
```suggestion
            "description": "The unique identifier for your dandiset. Will automatically determine whether to upload to the main DANDI archive or the development staging server."
```
Slight rephrase suggestion

**Comment by CodyCBakerPhD** [comment]
This should probably say '2 fields', right?

image

**Comment by CodyCBakerPhD** [comment]
Aside from that one last minor display item, I tested this out on both the main archive and staging as well as the standalone page and the guided mode

Seems to work well!

**Comment by garrettmflynn** [comment]
Good catch. Just fixed

**Review by CodyCBakerPhD — APPROVED** [review]

**Pull request merged**